src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: scale 0.3s;
+
+  &:hover {
+    scale: 1.2;
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 40px 0 0;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: 0%;
+    color: $main-card-text-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0%;
+    color: $secondary-card-text-color;
+  }
+
+  &__rates {
+    margin: 16px 0 0;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-reviews {
+      margin: 0;
+
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 16px;
+      letter-spacing: 0%;
+      color: $main-card-text-color;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 0;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      margin: 0;
+
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      letter-spacing: 0%;
+      color: $secondary-card-text-color;
+    }
+
+    &-cost {
+      margin: 0;
+
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      letter-spacing: 0%;
```

Comment:
As with the other `letter-spacing` properties set to `0`, this one is also unnecessary and can be removed.